src/Sarif.Multitool/TransformCommand.cs

```diff
@@ -78,7 +78,9 @@ public int Run(TransformOptions transformOptions)
                 {
                     if (inputVersion == "1.0.0")
                     {
-                        _fileSystem.WriteAllText(fileName, _fileSystem.ReadAllText(inputFilePath));
+                        SarifLogVersionOne logV1 = ReadSarifFile<SarifLogVersionOne>(_fileSystem, transformOptions.InputFilePath, SarifContractResolverVersionOne.Instance);
+                        logV1.SchemaUri = SarifVersion.OneZeroZero.ConvertToSchemaUri();
+                        WriteSarifFile(_fileSystem, logV1, fileName, formatting, SarifContractResolverVersionOne.Instance);
```

Comment:
so, now whenever someone converts to v1 the url shall become: [https://schemastore.azurewebsites.net/schemas/json/sarif-1.0.0.json](https://schemastore.azurewebsites.net/schemas/json/sarif-1.0.0.json) (hence this transformation)